conference.js

```diff
@@ -2387,7 +2400,19 @@ export default {
         APP.store.dispatch(conferenceJoined(room));
 
         APP.UI.mucJoined();
-        APP.API.notifyConferenceJoined(APP.conference.roomName);
+        const displayName = APP.settings.getDisplayName();
+
+        APP.API.notifyConferenceJoined(
+            this.roomName,
+            this._room.myUserId(),
+            {
+                displayName,
+                formattedDisplayName: formatDisplayName(
+                    displayName,
+                    interfaceConfig.DEFAULT_LOCAL_DISPLAY_NAME),
+                avatar: APP.UI.getAvatarUrl()
```

Comment:
Maybe the key should be avatarUrl to be consistent with the method name and what's kept in the store.

Reply:
Now I checked and in the store and the whole react folder we are using avatarURL. So what about renaming it to avatarURL?

Reply:
Sure, sounds good.

Reply:
Fixed!